gramex/handlers/modelhandler.py

```diff
@@ -6,16 +6,44 @@
 import gramex.data
 from gramex.handlers import BaseHandler
 import tornado.escape
+from io import BytesIO
+
+_model_cache = {}
 
 
 class ModelHandler(BaseHandler):
     '''
     Allows users to create API endpoints to train/test models exposed through Scikit-Learn.
     TODO: support Scikit-Learn Pipelines for data transformations.
     '''
+    def _load_keras_model(self, path):
+        """Load a Keras model into a local cache.
+
+        Parameters
+        ----------
+        path : str
+            Path to a Keras model.
+        """
+        model = _model_cache.get(path, False)
+        if not model:
+            _model_cache[path] = gramex.cache.open(path)
+        return _model_cache[path]
+
     @classmethod
     def setup(cls, path, **kwargs):
+        if os.path.splitext(path)[-1] == '.h5':
+            from tensorflow.keras.models import load_model
+            gramex.cache.open_callback['h5'] = load_model
+            cls.model_type = "keras"
+            cls._model_loader = cls._load_keras_model
+        else:
+            cls.model_type = "sklearn"
+            cls._model_loader = staticmethod(lambda x: gramex.cache.open(x, gramex.ml.load))
```

Comment:
@jaidevd -- should `gramex.ml.load(..., engine='keras')` handle both keras ans sklearn instead here?

Reply:
`gramex.ml.load` is just `joblib.load` - should we add an explicit `gramex.ml.load` that does this (takes the `engine` kwarg and decides what to do)?